docs/content/migration.md

```diff
@@ -0,0 +1,503 @@
+
+AngularJS Material offers a number of _run-time_ theming features. AngularJS Material's
+[theming system][106] is built on the generation of CSS. Details can be found in the
+[Under the hood][157] section of the theming guide. If many themes are generated at
+compile-time, the resulting app will put a strain on low-bandwidth connections. If many themes are
+generated by JavaScript at run-time, the resulting app may generate high CPU and/or memory usage.
+
+Given the lessons learned from the AngularJS Material theming system, the team selected a more
+performant _compile-time_ approach. Angular Material performs theming with [Sass][170]. While this
+approach requires more up-front set-up, it completely removes all JavaScript execution cost
+associated with theming. The use of Sass also moves the theming system to a well-supported and
+broadly-understood toolchain.
+
+#### AngularJS Material Compile-time Theming
+Features include:
+
+- [Pre-built palettes][150] based on the [Material Design Palettes][148]
+- [Custom palette creation][151] including definition of hues and contrast colors
+- [Custom theme creation][152] including definition of color intentions and whether it is a dark or
+  light theme
+- [Generation of multiple themes][153] at build time
+- [Compilation of static theme files][176]
+- Use specific themes on specific elements ([overriding the default theme][155])
+- Modify the [browser header and system status bar colors][149] on mobile at compile-time
+
+#### AngularJS Material Run-time Theming
+Features include:
+
+- [Definition and generation of themes][143] at run-time
+- [Change the application-wide default theme to another theme][154] at run-time
+- [Use dynamically changing themes for specific components][156] (overriding the default theme)
+- Modify the [browser header and system status bar colors][149] on mobile at run-time
+- [Dynamically apply `theme-palette-hue-opacity` color combinations][73] as CSS styles to your own
+  application-specific components
+
+#### Angular Material Compile-time Theming
+Features include:
+
+- [Pre-built palettes][163] based on the [Material Design Palettes][148]
+- Custom palette creation including definition of hues and contrast colors can be accomplished by
+  copying one of the [provided palettes][167], modifying it, and then using it in your custom theme
+  creation
+- [Custom theme creation][164] including definition of color intentions and whether it is a dark or
+  light theme
+- [Generation of multiple themes][165] at build time
+- [Theme customization for a set of component types][166] (i.e. `mat-button` and `mat-checkbox`)
+- Theme customization of a specific set of Angular Material elements based on CSS selectors using
+  Sass
+- Use Angular Material themes to [style application or library components][168]
+
+#### Run-time Theming in an Angular Material app
+
+Features include:
+- Angular's [Meta service][147] can be used to dynamically modify `<meta>` tags that affect browser
```

Comment:
Yes, this link points to the framework docs. This is only here because in AngularJS Material this is a feature of theming and included in the theming docs. Users doing to migration will want to know how to migrate this feature.